elasticdl/python/elasticdl/layers/embedding.py

```diff
@@ -44,6 +48,7 @@ def __init__(
         self.supports_masking = mask_zero
         self.input_length = input_length
         self.combiner = combiner
+        self.embedding_service_endpoint = (embedding_service_endpoint,)
```

Comment:
Why do we need to convert `embedding_service_endpoint` to a tuple?

Reply:
Done, remove it.